src/Gui/Viewer/Viewer.hpp

```diff
@@ -181,6 +181,13 @@ class RA_GUI_API Viewer : public WindowQt, public KeyMappingManageable<Viewer>
 
     void needUpdate();
 
+    /// Event sent after a mouse press event has been processed, but before emitting needUpdate()
+    void onMousePress( QMouseEvent* event );
```

Comment:
```suggestion
    [[deprecated]] void onMousePress( QMouseEvent* event );
```

Note that for deprecated method, some IDE will not automatically complement their name from any prefix and will display the available method as strikethrough. This might discourage users to abuse from ready-to-be-removed features ;)